narwhal/types/src/primary.rs

```diff
@@ -367,10 +368,25 @@ impl Hash<{ crypto::DIGEST_LENGTH }> for BatchV2 {
     }
 }
 
+// Messages generated internally by Narwhal that are included in headers for sequencing.
+#[derive(Clone, Deserialize, MallocSizeOf, Serialize)]
+pub enum SystemMessage {
+    DkgMessage(
+        fastcrypto_tbls::dkg::Message<
+            <ThresholdBls12381MinSig as ThresholdBls>::Public,
+            <ThresholdBls12381MinSig as ThresholdBls>::Public,
```

Comment:
BEACON_PK_G & BEACON_ENC_G?

Reply:
Those are local to the StateHandler, and I didn't want to make another alias here when it's only used this once